inputs/orchestrator_inner:6.json

```diff
@@ -33,6 +33,9 @@
         "inspect_only": true
       }
     },
+    {
+      "name": "hide_files"
```

Comment:
I don't see why we need to run this on the orchestrator build. We need it in the worker build only.

Reply:
fixed